### PR TITLE
Misc queue from 2018 (sorry, lost sources)

### PR DESCRIPTION
**Pull request by Gelma**

### CODE REVIEW

**Comment by larsoner** [comment]
merge conflict

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -5735,6 +5737,7 @@ diposed->disposed, deposed,
 diposing->disposing, deposing,
 dipthong->diphthong
 dipthongs->diphthongs
+dircet->dircet
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: This is a self-replacement, I think the correction should be direct.

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -5745,6 +5748,7 @@ direclty->directly
 direcly->directly
 direcories->directories
 direcory->directory
+direcotory->directory
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Add the plural

File: codespell_lib/data/dictionary.txt [hunk 3]
```diff
@@ -10323,6 +10328,7 @@ liasons->liaisons
 libarary->library
 libaries->libraries
 libary->library
+libatomic-ops->libatomic_ops
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Specific

File: codespell_lib/data/dictionary.txt [hunk 4]
```diff
@@ -15031,6 +15039,7 @@ santizes->sanitizes
 santizing->sanitizing
 sargant->sergeant
 sargeant->sergeant
+sarting->starting
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Suggest sorting too

File: codespell_lib/data/dictionary.txt (hunk 1 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Done

File: codespell_lib/data/dictionary.txt (hunk 2 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Done

File: codespell_lib/data/dictionary.txt (hunk 3 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Removed

File: codespell_lib/data/dictionary.txt (hunk 4 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Done

**Comment by Gelma** [comment]
I'm going to upload new version.

**Pull request merged**